changelogs/fragments/regex_search_return_empty_str.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

I would mark this as a breaking change (and not backport it), since it could be that some people depend on this behavior and this change will break their workflow.
```suggestion
breaking_changes:
```